test/libyul/yulSyntaxTests/instructions.yul

```diff
@@ -0,0 +1,3 @@
+{ pop }
+// ----
+// ParserError 2314: (6-7): Expected '(' but got '}'
```

Comment:
That's surprisingly not the same error message as in the old test...

Reply:
Changed the dialect - that did the trick.